### ISSUE TITLE
Relative links in root get broken after navigation (GitHub Pages)

### ISSUE THREAD

**Issue by prathwik0** [rat-tub/frontend-components#12]
### Try it yourself

- Goto [frontend-components/positioning/static](https://rat-tub.github.io/frontend-components/positioning/static)
- Navigate to Home
- Now the relative links in root page no longer work

### Images for reference

<img height="200" alt="frontend-components/positioning/static" src="https://github.com/rat-tub/frontend-components/assets/64214685/0e514d11-2069-499c-ae51-56fbaa0c32f4">

<img height="200" alt="404" src="https://github.com/rat-tub/frontend-components/assets/64214685/c799352b-6ac9-4c31-ae67-ffa13c4ef815">

**Comment by 05hiteshbansal** [comment]
i m interested in this issue can you assign me

**Comment by prathwik0** [comment]
Yeah! Would be grateful for any help. One way to fix is is to use absolute links 

```svelte
<script lang="ts">
  import { base } from "$app/paths";
</script>

# we can write absolute links like this
<a href="{base}/">...</a>
```

But I would like to know what causes this behaviour and if this could be fixed while still using relative links.
